### PR TITLE
[Command] Add import command

### LINKED ISSUE #86: [Command] Add import command
e.g. /import tut35

### PR DESCRIPTION
**Pull request by kevinmingtarja**
Closes #86 

Recording:

https://user-images.githubusercontent.com/69668484/137708900-7df463fc-750b-4184-88ef-6472353454cc.mov

### CODE REVIEW

File: src/main/java/seedu/address/logic/commands/ImportCommand.java [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+
+    public static final String COMMAND_WORD = "import";
+
+    public static final String MESSAGE_USAGE = COMMAND_WORD
+            + ": Imports a file identified by the given filename.\n"
+            + "Parameters: FILENAME\n"
+            + "Example: " + COMMAND_WORD + " t35";
+
+    public static final String MESSAGE_SUCCESS = "File successfully imported. %1$d new contacts added";
+    public static final String MESSAGE_INCORRECT_FORMAT = "Data file not in the correct format. Please try again";
```
Comment (hsiaotingluv) [inline src/main/java/seedu/address/logic/commands/ImportCommand.java:34]: what is the correct format? should we specify more instruction for the user?

File: src/main/java/seedu/address/logic/parser/ImportCommandParser.java [hunk 2]
```diff
@@ -0,0 +1,20 @@
+package seedu.address.logic.parser;
+
+import seedu.address.logic.commands.ImportCommand;
+import seedu.address.logic.parser.exceptions.ParseException;
+
+/**
+ * Parses input arguments and creates a new DeleteCommand object
```
Comment (hsiaotingluv) [inline src/main/java/seedu/address/logic/parser/ImportCommandParser.java:7]: perhaps update the DeleteCommand to ImportCommand!

File: src/test/java/seedu/address/logic/commands/AddCommandTest.java [hunk 3]
```diff
@@ -118,6 +118,11 @@ public void setAddressBook(ReadOnlyAddressBook newData) {
             throw new AssertionError("This method should not be called.");
         }
 
+        @Override
```
Comment (hsiaotingluv) [inline src/test/java/seedu/address/logic/commands/AddCommandTest.java:121]: not a major problem for now but you might want to try adding new import command test cases e.g. import empty file, wrong paths etc :))

**Review by choonyongchan — APPROVED** [review]
LGTM. Just a very minor design consideration, perhaps the format of the new Test class can follow the other Test classes, specifically declaring constants for 'input.txt', 'import.json'. I'll look through it later :) Great work

**Pull request merged**